src/plugins/data/common/search/session/utils.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { createHash } from 'crypto';
```

Comment:
Seems there is already a trimmed down version of exactly what we need in core: `src/core/public/utils/crypto/sha256.ts`

Since this code is in `/common` and has a risk of getting into UI, should we consider using a trimmed down version from `core` now? 
(Small caveat is that we will need to move that util to `common` first: `src/core/public/utils/crypto/sha256.ts` -> `src/core/common/utils/crypto/sha256.ts`)